sklearn/svm/tests/test_svm.py

```diff
@@ -353,6 +353,7 @@ def test_decision_function_shape():
     assert_equal(dec.shape, (len(X_train), 10))
 
 
+@ignore_warnings
```

Comment:
I feel the function should also check that decision_function is the same as ``predict``, and test should be renamed accordingly. Or is there another test for ``SVR.predict``? When ``decision_function`` gets removed we still want to compare ``predict`` against the manual computation.